mesonbuild/build.py

```diff
@@ -1301,6 +1301,16 @@ def add_deps(self, deps):
         for dep in deps:
             if dep in self.added_deps:
                 continue
+
+            dep_d_features = dep.get_d_features()
+
+            for feat in ('versions', 'import_dirs'):
+                if feat in dep_d_features:
+                    if feat in self.d_features:
+                        self.d_features[feat].extend(dep_d_features[feat])
+                    else:
+                        self.d_features[feat] = dep_d_features[feat]
```

Comment:
Offtopic: I have an irrational dislike of the token `feat`, because it makes me think this: https://www.dictionary.com/browse/feat

> a noteworthy or extraordinary act or achievement, usually displaying boldness, skill, etc.

I wonder if maybe we could use `for feature in ('versions', 'import_dirs'):` instead?

Reply:
sure :smile: I didn't know this word!